### ISSUE TITLE
GetText.NET.Extractor: POT files’s references are Unix paths or Windows?

### ISSUE THREAD

**Issue by mcka-dev** [perpetualKid/GetText.NET#19]
Hi,

Source:
```
using System;
using System.Windows.Forms;

namespace HelloApp
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent()

            // Must be double "Text1"
            button1.Text = "Text1";
            button1.Text = "Text1";
        }
    }
}
```
**Actual result in GetText.NET.Extractor v0.9.6.0:**
![изображение](https://user-images.githubusercontent.com/16173503/99142066-89792300-2662-11eb-9a61-4abac65a71ec.png)

If you then open this in PoEdit and click save, we will get:
**Actual result in PoEdit 2.4.2:** 
![изображение](https://user-images.githubusercontent.com/16173503/99142174-d1e51080-2663-11eb-9374-0f24a398ea18.png)

References is broken (((

Which path is correct if there are spaces in the path?

`#: .\Path with spaces\2.cs:13`
or Unix paths, not Windows ones
`#: ./Path with spaces/2.cs:13`

**Comment by perpetualKid** [comment]
This seems to be an issue in PoEdit, either style (forward slash or backslash path separator) works well on Windows systems, and PoEdit can open the references just fine even with blanks in the path.

The issue only happends when PoEdit saves the file.

Doesn't even work to quote the path like #: ".\Path with spaces\2.cs":13, PoEdit still breaks on the second reference when saved in PoEdit, but furthermore can't open the path when quoted. 

Therefore I'd suggest to open an issue with PoEdit.

If you have a reference or documentation if/how path with blanks shall be quoted, I'm open to adapt, else I see this working well from Extractor-side but somehow failing  in PoEdit.

**Comment by mcka-dev** [comment]
I got a response from Václav Slavík (Poedit):

> You observed a design shortcoming in GNU gettext (not Poedit) which uses whitespace as delimiters for multiple source code references (under the not-entirely-unreasonable assumption of source code using sane paths, i.e. without spaces) as well as a bug in [GetText.NET Extractor](https://github.com/perpetualKid/GetText.NET) (PO files’s references are Unix paths, not Windows ones).
>
> There really isn’t anything Poedit can do about it. GNU gettext might, but I suspect they’re opinion would be pragmatic: just don’t use spaces in source  code paths, it’s just asking for problems everywhere. And the PO spec is too old and too widespread to be amended to handle this edge case. 
>
> Regards, Václav

**Comment by perpetualKid** [comment]
I'm thinking to make the path separator configurable via command line option, but that wouldn't help anything with spaces in code path. Hence the pragmatical solution as recommended is just to avoid spaces in the source code path.

**Comment by perpetualKid** [comment]
available in 0.9.7 using commandline paramater -u or --unixstyle
